### PR TITLE
chore(tfhe): update boolean and shortint parameters

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs.rs [hunk 1]
```diff
@@ -2,40 +2,39 @@ use crate::core_crypto::prelude::*;
 use crate::shortint::ciphertext::MaxNoiseLevel;
```
Comment (IceTDrinker) [inline tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs.rs:2]: not sure we want to change those for now (multi bit analysis in progress)

**Comment by IceTDrinker** [comment]
we likely need to check with Sam first (and JB as the changes are relatively minor)

File: tfhe/src/boolean/parameters/mod.rs [hunk 2]
```diff
@@ -127,52 +127,52 @@ impl BooleanKeySwitchingParameters {
 /// $2^{-40}$. The secret keys generated with this parameter set are uniform binary.
 /// This parameter set allows to evaluate faster Boolean circuits than the `TFHE_LIB_PARAMETERS`
 /// one.
-pub const DEFAULT_PARAMETERS: BooleanParameters = BooleanParameters {
-    lwe_dimension: LweDimension(811),
+// p-fail = 2^-64.017, algorithmic cost ~ 67, 2-norm = 2.8284271247461903
+pub const DEFAULT_PARAMETERS_KS_PBS: BooleanParameters = BooleanParameters {
```
Comment (IceTDrinker) [inline tfhe/src/boolean/parameters/mod.rs:131]: keep the original order of parameters please

File: tfhe/src/boolean/parameters/params.rs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+/// This parameter set ensures 132-bits of security, and a probability of error is upper-bounded by
+/// $2^{-64}$. The secret keys generated with this parameter set are uniform binary.
+/// This parameter set allows to evaluate faster Boolean circuits than the `TFHE_LIB_PARAMETERS`
+/// one.
+// p-fail = 2^-64.344, algorithmic cost ~ 75, 2-norm = 2.8284271247461903
+pub const DEFAULT_PARAMETERS: BooleanParameters = BooleanParameters {
+    lwe_dimension: LweDimension(805),
+    glwe_dimension: GlweDimension(3),
+    polynomial_size: PolynomialSize(512),
+    #[allow(clippy::excessive_precision)]
```
Comment (IceTDrinker) [inline tfhe/src/boolean/parameters/params.rs:14]: this still required ?

**Review by IceTDrinker — APPROVED** [review]

File: tfhe/src/boolean/parameters/params.rs (hunk 3 above)
Comment (soonum) [reply tfhe/src/boolean/parameters/params.rs:14]: Yes it is.

**Pull request merged**